### DISCUSSION TITLE
Source Code Not Updating Automatially

### DISCUSSION THREAD

**Discussion by TegoTech** [vercel/community#7379] [Feature Question]
### Question

Hi, I've been a Vercel user for a few years. First, I want to say thank you, as this is one of my favorite technical products I have ever used. You guys rock.

The issue I am having is that for some reason my source code is not currently updating to Vercel. So, I have been making changes on GitHub, but those changes are not appearing on the website. I went to my Vercel login and redeployed, but the deployment is pulling the old source code. 

My question is about the automatic update feature. I am wondering why the code is being pulled from an old source, rather than GitHub. I want to find a way to update the code, or connect my Vercel and GitHub, such that the deployment pulls the updated code. Any advice, suggestions, or help would be greatly appreciated.

**Comment by TegoTech** [comment]
This has resolved. It took about an hour to update. Thanks!

**Reply by pawlean** [reply]
Thanks for updating this, @TegoTech! 👍🏼

**Comment by citruscai** [comment]
I had same problem. I know it was addressed and apparently resolved and i thought it finally did when my deployment finally updated but now  i made 3 new pushes and it hasn't updated. is it just slow at the moment?

**Reply by pawlean** [reply]
Hi @citruscai! Are you still seeing this?

[a comment by an automated account is not shown]
